### DISCUSSION TITLE
Link the checkin with work item (task,bug...)

### DISCUSSION THREAD

**Discussion by mohammedmax** [jessehouwing/vsts-tfvc-tasks#115] [Ideas]
Possibility of linking the task with changes

**Comment by jessehouwing** [comment]
I am not adding any new functionality into these tasks. They're 'feature complete' just like TFVC is. 

I no longer have my own repos nor client repos in TFVC and major improvements would require too much effort on my side to ensure nothing breaks for the existing users.
